examples/context-propagation/kotlin/src/main/kotlin/example/armeria/contextpropagation/manual/Main.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package example.armeria.contextpropagation.manual
+
+import com.linecorp.armeria.client.WebClient
+import com.linecorp.armeria.common.HttpResponse
+import com.linecorp.armeria.common.HttpStatus
+import com.linecorp.armeria.server.Server
+import com.linecorp.armeria.server.ServiceRequestContext
+
+fun main(args: Array<String>) {
+    val backend = Server.builder()
+        .service("/square/{num}") { ctx: ServiceRequestContext, _ ->
+            val num = try {
+                ctx.pathParam("num")?.toLong()
+            } catch (e: NumberFormatException) {
```

Comment:
We're not bothering being safe about this check this in other examples, probably fine not to here (or otherwise we could update the other examples